packages/bfx-containers/src/functions/api.js

```diff
@@ -14,7 +14,7 @@ const DEFAULT_REQUEST_HEADERS = {
 }
 
 const createApiInstance = (baseURL) => axios.create({
-  baseURL: `${corsProxyUrl}https:${baseURL}`,
+  baseURL: `${corsProxyUrl}http:${baseURL}`,
```

Comment:
I thinks its not good idea, to use unsecured connection for all ufx-ui components, as this change will affect them
If `http` is required for in bfx-hf-ui, could you customise with env like REACT_APP_UFX_HTTP_PROTOCOL please.